### PR TITLE
Check and fix typos in CI

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind ci

#### What this PR does / why we need it:
Check and fix typos in CI.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by sohankunkerkar — APPROVED** [review]
/lgtm

File: docs/crio.conf.5.md [hunk 1]
```diff
@@ -241,7 +241,7 @@ the container runtime configuration.
   This option is deprecated. The Kubelet flag `--container-log-max-size` should be used instead.
 
 **log_to_journald**=false
-  Whether container output should be logged to journald in addition to the kuberentes log file.
+  Whether container output should be logged to journald in addition to the kubernetes log file.
```
Comment (TomSweeneyRedHat) [inline docs/crio.conf.5.md:244]: I think: 
```suggestion
  Whether container output should be logged to journald in addition to the Kubernetes log file.
```

File: pkg/config/template.go [hunk 2]
```diff
@@ -1017,7 +1017,7 @@ const templateStringCrioRuntimeLogSizeMax = `# Maximum sized allowed for the con
 
 `
 
-const templateStringCrioRuntimeLogToJournald = `# Whether container output should be logged to journald in addition to the kuberentes log file
+const templateStringCrioRuntimeLogToJournald = `# Whether container output should be logged to journald in addition to the kubernetes log file
```
Comment (TomSweeneyRedHat) [inline pkg/config/template.go:1020]:
```suggestion
const templateStringCrioRuntimeLogToJournald = `# Whether container output should be logged to journald in addition to the Kubernetes log file
```

File: test/testdata/config/config-v1.17.0.toml [hunk 3]
```diff
@@ -186,7 +186,7 @@ pids_limit = 1024
 # limit is never exceeded.
 log_size_max = -1
 
-# Whether container output should be logged to journald in addition to the kuberentes log file
+# Whether container output should be logged to journald in addition to the kubernetes log file
```
Comment (TomSweeneyRedHat) [inline test/testdata/config/config-v1.17.0.toml:189]:
```suggestion
# Whether container output should be logged to journald in addition to the Kubernetes log file
```

**Review by TomSweeneyRedHat — COMMENTED** [review]
I'm pretty sure we should capitalize the first letter in Kubernetes, especially on anything user-facing.  Otherwise LGTM.

**Comment by umohnani8** [comment]
LGTM
/retest

**Pull request merged**